os/kernel/binary_manager/binary_manager_getinfo.c

```diff
@@ -297,3 +217,53 @@ void binary_manager_get_info_all(int requester_pid)
 #endif
 	binary_manager_send_response(q_name, &response_msg, sizeof(binmgr_getinfo_all_response_t));
 }
+
+int binary_manager_get_download_path(int requester_pid, char *bin_name)
```

Comment:
There is already `binary_manager_get_download_path` API in framework.
Please change one of the name.
It is not good to have same function name.